File: Python/99problems.py
import sys

info = sys.stdin.readline()
tall = int(info)

if tall < 149:
    print(99)
else:
    lastTwo = info[len(info) - 3:]
    if int(lastTwo) >= 49:
        print(tall + (99 - int(lastTwo)))
    else:
        print(tall - (int(lastTwo) + 1))

File: Kotlin/src/99Problems.kt
fun main() {
    val info = readLine()!!
    val tall = info.toInt()

    if (tall < 149)
        println(99)
    else {
        val lastTwo = (info[info.length - 2] + "" + info[info.length - 1])
        if (lastTwo.toInt() >= 49) {
            println(tall + (99 - lastTwo.toInt()))
        } else
            println(tall - (lastTwo.toInt() + 1))
    }
}